components/org.wso2.carbon.identity.oauth.scope.endpoint/src/main/java/org/wso2/carbon/identity/oauth/scope/endpoint/impl/ScopesApiServiceImpl.java

```diff
@@ -233,4 +246,48 @@ public Response deleteScope(String name) {
         }
         return Response.status(Response.Status.OK).build();
     }
+
+    /**
+     * Build the complete URI to the location of the resource.
+     * Ex: https://localhost:9443/t/<tenant-domain>/identity/oauth2/v1.0/scopes/name/<scope_name>
+     *
+     * @param name of the scope.
+     * @return Fully qualified and complete URI.
+     */
+    private static URI buildURIForHeader(String name) {
```

Comment:
Can we change the parameter name to something like `scopeName`

Reply:
Changed 'name' to 'scopeName'